client/src/components/MapLegend/index.tsx

```diff
@@ -0,0 +1,130 @@
+import {defineMessages} from 'react-intl';
+import {Tooltip} from '@trussworks/react-uswds';
+
+import * as styles from './mapLegend.module.scss';
+import * as constants from '../../data/constants';
+
+// @ts-ignore
+import infoIcon from '/node_modules/uswds/dist/img/usa-icons/info_outline.svg';
+
+// Todo VS: This information will be used in the re-design of the tool-tip
+// const getToolTipContent = (type:string) => {
+//   const intl = useIntl();
+//   const messages = defineMessages({
+//     priorityHeader: {
+//       id: 'tooltip.info.priority.header',
+//       defaultMessage: constants.PRIORITIZED_COMMUNITY,
+//       description: 'the header of the prioritized community tooltip',
+//     },
+//     thresholdHeader: {
+//       id: 'tooltip.info.threshold.header',
+//       defaultMessage: constants.THRESHOLD_COMMUNITY,
+//       description: 'the header of the threshold community tooltip',
+//     },
+//     priorityText: {
+//       id: 'tooltip.info.priority.text',
+//       defaultMessage: 'A prioritized community is one that has a cumulative index score of Xth ' +
+//       'percentile and above. 40% of the benefits from investments outlined by the ' +
+//       'Justice40 Initiative should go to prioritized communities.',
+//       description: 'the text of the prioritized community tooltip',
+//     },
+//     thresholdText: {
+//       id: 'tooltip.info.threshold.text',
+//       defaultMessage: 'Communities with a cumulative index score between Y - X.99th percentile are ' +
+//      'considered threshold communities. While these communities are currently not considered a ' +
+//      'prioritized community, this may change based on updates to the scoring method.',
+//       description: 'the text of the threshold community tooltip',
+//     },
+//   });
+
+//   return (type === 'prioritized') ?
+//     (
+//     <div>
+//       <h2>{intl.formatMessage(messages.priorityHeader)}</h2>
+//       <p className={styles.legendTooltipText}>{intl.formatMessage(messages.priorityText)}</p>
+//     </div>
+//     ) :
+//     (
+//       <div>
+//         <h2>{intl.formatMessage(messages.thresholdHeader)}</h2>
+//         <p className={styles.legendTooltipText}>{intl.formatMessage(messages.thresholdText)}</p>
+//       </div>
+//     );
+// };
+
+const MapLegend = () => {
+  const intl = useIntl();
+  const messages = defineMessages({
+    priorityLabel: {
+      id: 'legend.info.priority.label',
+      defaultMessage: constants.PRIORITIZED_COMMUNITY,
+      description: 'the label of the prioritized community legend',
+    },
+    thresholdLabel: {
+      id: 'legend.info.threshold.label',
+      defaultMessage: constants.THRESHOLD_COMMUNITY,
+      description: 'the label of the threshold community legend',
+    },
+  });
+
+  // Type definitions required for @trussworks tooltip. This type defines the div that wraps the icon.
+  // This allows to pass children and other attributes.
+  type IconWrapperProps = React.PropsWithChildren<{
```

Comment:
We can't just do 
```js
const text = "Communities that have cumulative 
            index score of Xth percentile
            and above"
<Tooltip position="left" label={text}>
      {text}
    </Tooltip>
``` ?

Reply:
more examples [here](https://github.com/trussworks/react-uswds/blob/main/src/components/Tooltip/Tooltop.stories.tsx)

Reply:
I don't think we can. I'm sorry Nat, I know you've brought this up a few times, but I'm not seeing how to use the default version. Thanks for being so patient! 

This code renders this:

![Screen Shot 2021-08-06 at 3 07 25 PM](https://user-images.githubusercontent.com/86254807/128575603-debe4b8d-deea-45d7-8bf6-c3f9377ffd43.png)

My current understanding leads me to believe that there are a few issues with this approach:

1. It seems to me that the target of the tooltip is the default blue button (shown above), in our case we want it to be the icon (something we set). Do you know how to change what's targeted using the default case you've mentioned? Placing the icon as a child of `<Tooltip>` still renders the button.  I believe that's why they have a custom tooltip component for this very reason. But maybe I'm missing something? If my understanding is correct, this is a deal breaker. 
2. Assuming bullet 1 is correct, having the label be a `text` variable doesn't allow us to control its width, which gets cut off on mobile. The tooltip doesn't seem to scale for mobile. Might be some lower level uswds classes that could fix this, however bullet 1 still dominates.

Looking forward to hearing your thoughts!

Reply:
ah ok, I had misunderstood what was what here, thank you for the explanation -- I was thinking we could just treat the image as a button, which in this case it effectively is (and in fact for accessibility reasons we will need it to be), but didn't realize it came along with this other button-styling baggage. 

Looks like this way works to wrap the `img` in an `a` and do the right thing from an accessibility perspective